src/components/dialogs/parameters/non-evacuated-energy/non-evacuated-energy-parameters-selector.tsx

```diff
@@ -14,16 +14,17 @@ import { TabPanel } from '../parameters';
 import { useCreateRowDataSensi } from '../../../../hooks/use-create-row-data-sensi';
 import * as nonEvacuatedEnergyParam from './columns-definitions';
 import {
+    NonEvacuatedEnergyContingencies,
     NonEvacuatedEnergyGenerationStages,
-    NonEvacuatedEnergyStagesSelection,
     NonEvacuatedEnergyGeneratorsCappings,
     NonEvacuatedEnergyMonitoredBranches,
-    NonEvacuatedEnergyContingencies,
+    NonEvacuatedEnergyStagesSelection,
     TAB_VALUES,
 } from './columns-definitions';
 import SensitivityTable from '../sensi/sensitivity-table';
 import GeneratorsCappingsThreshold from './generators-cappings-threshold';
 import { IColumnsDef } from '../sensi/columns-definitions';
+import { parametersStyles } from '@gridsuite/commons-ui';
```

Comment:
You add a usage of something marked deprecated here.

Reply:
Note that it is a quick fix, we will rework the layout of all params at the same time in another PR..., so I reuse this constant to facilitate the reactorization..